mbta.js

```diff
@@ -0,0 +1,83 @@
+
+const subway_lines = {
+    Red: [
+      'South Station',
+      'Park Street',
+      'Kendall',
+      'Central',
+      'Harvard',
+      'Porter',
+      'Davis',
+      'Alewife'
+    ],   length_of_Red:function(){return this.Red.length; },
+Green: [
+      'Government Center',
+      'Park Street',
+      'Boylston',
+      'Arlington',
+      'Copley',
+      'Hynes',
+      'Kenmore'
+],
+      length_of_Green:function(){return this.Green.length; },
+    Orange: [
+      'North Station',
+      'Haymarket',
+      'Park Street',
+      'State',
+      'Downtown Crossing',
+      'Chinatown',
+      'Back Bay',
+      'Forest Hills'
+    ], length_of_Orange:function(){
+    return this.Orange.length;
+      },
+  }
+
+function stopsBetweenStations(startLine, startStation, endLine, endStation){
+
+ let start= subway_lines[startLine].indexOf(startStation);
+ let end= subway_lines[endLine].indexOf(endStation);
+
+ console.log(`starts in ${startLine} Line ,${startStation} Station`);
+ console.log(`ends with ${endLine} Line ,${endStation} Station`);
+
+ if (startLine!= "Red" && startLine!= "Green" &&startLine!= "Orange" ) 
+    console.log("please enter a valid line");
+else if (endLine!= "Red" && endLine!= "Green" && endLine!= "Orange" ) 
+    console.log("please enter a valid line");
+    
+if(startLine==endLine&&startStation==endStation)
```

Comment:
The first condition is correct, but the second condition isn't needed. If that start and end station are the same then why is the user riding the subway?
If you remove the 2nd condition your code for one line journeys will be accurate.